### DISCUSSION TITLE
Question: two version of crate Bincode used

### DISCUSSION THREAD

**Discussion by dantengsky** [databendlabs/databend#4358] [Q&A] [answered]
**Summary**

Currently, there are two different versions of crate `bincode` used, shall we pick just one version of them?

one in query:

https://github.com/datafuselabs/databend/blob/45d9bd7cd70afbe1d19f6ba93a78a0950a1d7d03/query/Cargo.toml#L67

the other one in common-io:

https://github.com/datafuselabs/databend/blob/45d9bd7cd70afbe1d19f6ba93a78a0950a1d7d03/common/io/Cargo.toml#L20

**Comment by sundy-li** [comment, marked as answer]
Oh, we shall use the latter, we have some modification there.

**Comment by dantengsky** [comment]
> Oh, we shall use the latter, we have some modification there.

got it
